Doc/library/test.rst

```diff
@@ -928,8 +928,16 @@ The :mod:`test.support` module defines the following functions:
    .. versionadded:: 3.10
 
 
+.. function:: check_disallow_instantiation(test_case, tp, *args, **kwds)
+
+   Assert that type *tp* cannot be instantiated using *args* and *kwds*.
+
+   .. versionadded:: 3.11
```

Comment:
This is wrong, it should be 3.10. Once this PR is merged, please also fix the doc in the main branch.

Reply:
Oops, thanks

Reply:
Fixed with eb8d106cf3917cb89c823bf08859ebd640408e57